config/initializers/devise.rb

```diff
@@ -250,7 +250,16 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :shibboleth,
+                  # TODO: mappings will need to change when authenticating against IST's SAML
+                  uid_field: 'eppn',
```

Comment:
Yes.

We could change it slightly to `request.call('uid') || Rails.env.development && etc` to make it clearer that the second option is only expected in dev.